docs/source/interactive/autoawait.rst

```diff
@@ -127,7 +127,7 @@ Effects on IPython.embed()
 IPython core being asynchronous, the use of ``IPython.embed()`` will now require
 a loop to run. By default IPython will use a fake coroutine runner which should
 allow ``IPython.embed()`` to be nested. Though this will prevent usage of the
-``autoawait`` feature when using IPython embed. 
+:magic:`autoawait` feature when using IPython embed. 
```

Comment:
This one has no leading `%` but others do. Presumably only one of those is right, so they should be made consistent.

Otherwise 👍